File: CEMEP/Projeto_Rubia_2018/Projeto_Rubia_2018/RUBIAPPLICATION/RUBIAPPLICATION/Form11.vb
Public Class Frmcadlocatarios


    Private Sub LocatárioBindingNavigatorSaveItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles LocatárioBindingNavigatorSaveItem.Click
        Try
            Me.Validate()
            Me.LocatárioBindingSource.EndEdit()
            Me.TableAdapterManager.UpdateAll(Me.BDSystem1DataSet)

            MessageBox.Show("Registro salvo com sucesso", "informa", MessageBoxButtons.OK, MessageBoxIcon.Information)
        Catch
            MessageBox.Show("Falha ao salvar o registro", "informa", MessageBoxButtons.OK, MessageBoxIcon.Information)
        End Try
       
    End Sub

    Private Sub Form11_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        'TODO: This line of code loads data into the 'BDSystem1DataSet.Locatário' table. You can move, or remove it, as needed.
        Me.LocatárioTableAdapter.Fill(Me.BDSystem1DataSet.Locatário)

    End Sub

    Private Sub BindingNavigatorDeleteItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles BindingNavigatorDeleteItem.Click
        If MessageBox.Show("Deseja excluir esse registro?", "Exclusão", MessageBoxButtons.YesNo, MessageBoxIcon.Question) = Windows.Forms.DialogResult.Yes Then
            ''Se sim salvar, pegue a programação do salvar
            Me.Validate()
            Me.LocatárioBindingSource.EndEdit()
            Me.TableAdapterManager.UpdateAll(Me.BDSystem1DataSet)
            MessageBox.Show("Para concluir a exclusão, clique no BOTÃO SALVAR!")
        Else
            'Pegue a programacao do frmcadeditora_load que preenche o data set 
            Me.LocatárioTableAdapter.Fill(Me.BDSystem1DataSet.Locatário)

        End If
    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Close()
    End Sub

    Private Sub Frmcadlocatarios_KeyPress(ByVal sender As System.Object, ByVal e As System.Windows.Forms.KeyPressEventArgs) Handles MyBase.KeyPress
        If e.KeyChar = Microsoft.VisualBasic.ChrW(Keys.Return) Then
            SendKeys.Send("{Tab}")
            e.Handled = True
        End If
    End Sub
End Class

File: CEMEP/Industria_Metalurgica/Backup/Industria_Metalurgica/Form11.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace Industria_Metalurgica
{
    public partial class FrmFunc : Form
    {
        public FrmFunc()
        {
            InitializeComponent();
        }

        private void funcionáriosBindingNavigatorSaveItem_Click(object sender, EventArgs e)
        {
            this.Validate();
            this.funcionáriosBindingSource.EndEdit();
            this.tableAdapterManager.UpdateAll(this.industria_MetalurgicaDataSet);

        }

        private void FrmFunc_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'industria_MetalurgicaDataSet.Funcionários' table. You can move, or remove it, as needed.
            this.funcionáriosTableAdapter.Fill(this.industria_MetalurgicaDataSet.Funcionários);

        }

        private void button2_Click(object sender, EventArgs e)
        {
            Close(); 
        }

        private void FrmFunc_KeyPress(object sender, KeyPressEventArgs e)
        {
            if (e.KeyChar == (Char)13)
                SendKeys.Send("{Tab}");
        }

        private void button1_Click(object sender, EventArgs e)
        {
            OpenFileDialog Carregar = new OpenFileDialog();
         if (Carregar.ShowDialog() == DialogResult.OK)
        {
          Bitmap Bit = new Bitmap (Carregar.FileName);
          Carregar.DefaultExt = "*.JPG";
          Carregar.Filter = "Imagem (*.JPG) | *.JPG";
          capaPictureBox.Image = Bit; 
        }

        }
    }
}
